src/client.rs

```diff
@@ -274,17 +275,20 @@ impl Client {
         }
     }
 
-    /// Get the API keys from Meilisearch.
+    /// Get the API [Key]s from Meilisearch.
+    /// See the [meilisearch documentation](https://docs.meilisearch.com/reference/api/keys.html#get-all-keys).
+    ///
+    /// See also [Client::create_key] and [Client::get_key].
     ///
     /// # Example
     ///
     /// ```
-    /// # use meilisearch_sdk::{client::*, errors::Error};
+    /// # use meilisearch_sdk::{client::*, errors::Error, key::KeyBuilder};
     /// #
     /// # futures::executor::block_on(async move {
     /// let client = Client::new("http://localhost:7700", "masterKey");
     /// let keys = client.get_keys().await.unwrap();
-    /// assert_eq!(keys.len(), 2);
+    /// assert!(keys.len() > 2);
```

Comment:
I think this test is tightly coupled with the order the tests are run? By default 2 keys are created so it would only be > 2 if a key is created and still present before this test runs correct? Should it be `assert!(keys.len() >= 2);`? There is also the potential it is < 2 if a key was deleted and not re-added before this test runs.

Reply:
Oh yes good catch, I wanted to write `>= 2`!
> There is also the potential it is < 2 if a key was deleted and not re-added before this test runs.

I think for now we’re going to consider that we never delete the two default keys